src/footer/footer.macros.html

```diff
@@ -0,0 +1,59 @@
+{#
+  * @overview Macros for standard and simplified site Footers
+#}
+
+{% from '../icons/icons.macros.html' import icon as icon %}
+
+{#
+  * Renders a standard or simplified footer
+#}
+{% macro footer(simple=false, class=null, id=null) %}
+<footer class="footer{% if simple %} footer--simple{% endif %}{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %}>
+  <div class="footer-top r-inner">
+    {{ caller() }}
+  </div>
+  <div class="footer-bottom">
```

Comment:
@misscs If we are happy keeping the r- namespace for regions I don't mind going with that, although it feels sorta separate to the footer module when prefixed